Added quantlib functions to quant folder

def-mycroft opened the pull request:
okay, I just removed my old fork, created a new one and then added my quantlib modules; now I"m submitting this pr. I was able to pass the tests locally

def-mycroft committed:
Added quantlib functions to quant folder

Graeme22 commented on src/quant/test_models.py:
```diff
@@ -0,0 +1,24 @@
+from . import models as sc
+
+
+def test_model():
```
this should be in the `tests` directory at the top of the project hierarchy, that way it'll get run with `pytest`.

Graeme22 replied:
probably best to name the file `test_quant.py`

Graeme22 reviewed (changes requested):
In addition to the comment above, could you add QuantLib to the `requirements.txt` file?

def-mycroft commented:
Thanks for your comments, I'll move/rename the test file and update the
requirements files, maybe tonight I'm away from my computer now.

On Mon, May 17, 2021, 6:02 PM Graeme Holliday ***@***.***>
wrote:

> *@Graeme22* commented on this pull request.
> ------------------------------
>
> In src/quant/test_models.py
> <https://github.com/Graeme22/tastyworks-cli/pull/13#discussion_r633920202>
> :
>
> > @@ -0,0 +1,24 @@
> +"""Apply unit test to models"""
> +from . import models as sc
> +
> +
> +def test_model():
>
> probably best to name the file test_quant.py
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Graeme22/tastyworks-cli/pull/13#discussion_r633920202>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEY7CLDA73WJF2NQLXYWCZLTOGOBDANCNFSM45BJOJWA>
> .
>

def-mycroft commented:
Should I edit the same commit and force-push it or should I make a new
commit?

On Mon, May 17, 2021, 5:55 PM Graeme Holliday ***@***.***>
wrote:

> *@Graeme22* commented on this pull request.
> ------------------------------
>
> In src/quant/test_models.py
> <https://github.com/Graeme22/tastyworks-cli/pull/13#discussion_r633917678>
> :
>
> > @@ -0,0 +1,24 @@
> +"""Apply unit test to models"""
> +from . import models as sc
> +
> +
> +def test_model():
>
> this should be in the tests directory at the top of the project
> hierarchy, that way it'll get run with pytest.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Graeme22/tastyworks-cli/pull/13#pullrequestreview-661486223>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEY7CLDNEYNNPUA4E6TO4ODTOGNGLANCNFSM45BJOJWA>
> .
>

Graeme22 commented:
> Should I edit the same commit and force-push it or should I make a new
> commit?
> 
> On Mon, May 17, 2021, 5:55 PM Graeme Holliday ***@***.***>
> wrote:
> 
> > *@Graeme22* commented on this pull request.
> > ------------------------------
> >
> > In src/quant/test_models.py
> > <https://github.com/Graeme22/tastyworks-cli/pull/13#discussion_r633917678>
> > :
> >
> > > @@ -0,0 +1,24 @@
> > +"""Apply unit test to models"""
> > +from . import models as sc
> > +
> > +
> > +def test_model():
> >
> > this should be in the tests directory at the top of the project
> > hierarchy, that way it'll get run with pytest.
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/Graeme22/tastyworks-cli/pull/13#pullrequestreview-661486223>,
> > or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/AEY7CLDNEYNNPUA4E6TO4ODTOGNGLANCNFSM45BJOJWA>
> > .
> >
> 

A new one is usually better, and will retrigger the tests.

def-mycroft committed:
Updated requirements, moved quant test

Added QuantLib to requirements and moved src/quant/test_models.py to
tests/test_quant.py.

def-mycroft commented:
okay @Graeme22 , I made those changes. you have to review them and merge the pr, correct? let me know if there is anything else.
